### ISSUE TITLE
Repo of sample reports?

### ISSUE THREAD

**Issue by dobin** [google/sanitizers#931]
TL;DR: Is there a list of all possible reports ASAN creates? For asan-report-parser unit-testing purposes.

I'm fuzzing stuff and parsing the ASAN output to give an overview to the user, most importantly want to show the PC. My pipeline broke (Ubuntu 17.10) because it seems that newer clang versions added a different report format, called `memcpy-param-overlap` 
 (https://github.com/dobin/asanparser/blob/master/tests/memcpy-param-overlap.asan). Interestingly the main line does not show the PC anymore, which makes it is necessary to parse the stacktrace to extract PC. The ASAN output parser is therefore required to perform an exception handling `is-memcpy-param-overlap-report-or-not` (e.g. https://github.com/dobin/asanparser/blob/master/asanparser.py#L76)
The `memcpy-param-overlap` is also not documented in https://github.com/google/sanitizers/wiki/AddressSanitizer. 

In summary, i'm uncomfortable that my asan output parser will suddenly break again, and i'd like to test all possible cases. Is there an up to date repo of example report files, and if not, is it possible to create one?

I'm aware of https://github.com/google/sanitizers/issues/268 but for me personally a machine-readable format would be pretty useful.

**Comment by kcc** [comment]
There is no list other than the source, sorry. 
https://github.com/llvm-mirror/compiler-rt/blob/master/lib/asan/asan_errors.cc

`memcpy-param-overlap` existed for quite a while. 

The test suite is at https://github.com/llvm-mirror/compiler-rt/tree/master/test/asan but I am not sure if you can use it for your purposes. 

I remain opposed to implementing machine readable output format.

**Comment by dobin** [comment]
Alright thx!
